v0/destinations/klaviyo/transform.js

```diff
@@ -91,16 +92,11 @@ const addUserToList = async (message, traitsInfo, conf, destination) => {
   }
 };
 
```

Comment:
Here we should throw error if we are unable to add user to the list for customer visibility